### PR TITLE
Fix for position not working on mount

### PR DESCRIPTION
**Pull request by C-Higgins**
#32

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ let initialized = false;
 let soundManager;
 if (typeof window !== 'undefined') {
   soundManager = require('soundmanager2').soundManager;
-  
+  if (process.env.NODE_ENV === 'production') {
```
Comment (leoasis) [inline src/index.js]: Can we remove this? This doesn't belong to this particular fix, and also I believe we can do better, there's a discussion for this in https://github.com/leoasis/react-sound/issues/13#issuecomment-263689258

**Review by leoasis — CHANGES_REQUESTED** [review]
Awesome! thanks for the contribution!

Just made a comment for an unrelated change, but other than that, 👍

**Comment by C-Higgins** [comment]
Updated

**Comment by eveningkid** [comment]
Can't wait for the update, just got into the same issue! Thanks a lot guys. :)

**Review by leoasis — APPROVED** [review]

**Comment by leoasis** [comment]
Merged and published version 0.6.2 with this. Thanks for helping! https://github.com/leoasis/react-sound/releases/tag/v0.6.2

**Pull request merged**